vm/chef/cookbooks/influxdb/files/influxdb

```diff
@@ -14,44 +14,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-source /opt/c2d/c2d-utils || exit 1
+source /opt/c2d/influxdb-utils || exit 1
 
-api_port="$(get_attribute_value "api-port")"
-influx_url="http://localhost:${api_port}/ping"
+export INFLUXDB_ADMIN_PASSWORD="$(get_attribute_value "influxdb-admin-password")"
 
-export INFLUXDB_META_DIR="/var/lib/influxdb/meta"
-export INFLUXDB_DATA_DIR="/var/lib/influxdb/data"
-export INFLUXDB_DATA_WAL_DIR="/var/lib/influxdb/wal"
-export INFLUXDB_HTTP_AUTH_ENABLED=true
-export INFLUXDB_HTTP_BIND_ADDRESS=":${api_port}"
-export INFLUXDB_GRAPHITE_0_BIND_ADDRESS=":$(get_attribute_value "graphite-port")"
-
-if [[ "$(get_attribute_value "enable-graphite")" = True ]]; then
-  export INFLUXDB_GRAPHITE_0_ENABLED=true
-fi
-
-declare -a args=()
+# Run the CLI installer for InfluxDB
+declare -a args=(
+  --skip-verify \
+  --username admin \
+  --password "${INFLUXDB_ADMIN_PASSWORD}" \
+  --org Google_C2D_InfluxDB \
+  --bucket influxdb_bucket \
+  --retention 0 \
+  --force \
+  --name influxdb_config \
+)
 
 if [[ "$(get_attribute_value "enable-https")" = True ]]; then
   chown root:influxdb /etc/ssl/private
   chown root:influxdb /etc/ssl/private/c2d-temporary-self-signed-cert.key
   chmod 650 /etc/ssl/private
   chmod 640 /etc/ssl/private/c2d-temporary-self-signed-cert.key
-  influx_url="https://localhost:${api_port}/ping"
-  export INFLUXDB_HTTP_HTTPS_ENABLED=true
-  export INFLUXDB_HTTP_HTTPS_CERTIFICATE="/etc/ssl/certs/c2d-temporary-self-signed-cert.pem"
-  export INFLUXDB_HTTP_HTTPS_PRIVATE_KEY="/etc/ssl/private/c2d-temporary-self-signed-cert.key"
-  args+=(-ssl -unsafeSsl)
+  influx_url="https://localhost:8086"
+  args+=(--host="${influx_url}")
```

Comment:
if you're using the variable `influx_url` somewhere else, you put this array append out of the if condition.

if you're not using the variable, then just assign the raw value to the array append